packages/pods-core/src/builtin/NextjsExportPod.ts

```diff
@@ -270,7 +275,14 @@ export class NextjsExportPod extends ExportPod<NextjsExportConfig> {
     );
     const podDstPath = path.join(podDstDir, "notes.json");
     const podConfigDstPath = path.join(podDstDir, "dendron.json");
-    fs.writeJSONSync(podDstPath, payload, { encoding: "utf8", spaces: 2 });
+    fs.writeJSONSync(
+      podDstPath,
+      {
+        ...payload,
+        notes: mapObject(payload.notes, (_idx, { body: _body, ...note }) => note),
```

Comment:
can you leave a comment here that describes the function since its not immediately obvious? (eg. removes `body` from `notes`)